Services/DataBaseService/BrokerConsumers/DeleteUserConsumer.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace DataBaseService.BrokerConsumers
 {
-    public class DeleteUserConsumer : IConsumer<InternalDeleteUserRequest>
+    public class DeleteUserConsumer : IConsumer<InternalGetUserByIdRequest>
```

Comment:
If you decided to use same internal request model for get and delete, why do you have separate constructors?
I thought that if we have same request model, we decided to create different endpoints and then check endpoint in one consumer, which handles the request.
So seeing this now, I'm not even sure that it will work properly, because I can't see different endpoints configuration in DBS (but mb I'm wrong, then correct me)

Reply:
I don't really know what you mean, but I use different models

Reply:
Fixed